### ISSUE TITLE
Backspace removes two characters

### ISSUE THREAD

**Issue by mcatanzaro** [01org/ozone-wayland#419]
The backspace key removes two characters instead of one.

**Comment by michaelforney** [comment]
I am also observing this behavior.

**Comment by kalyankondapally** [comment]
Yes, I observe the same. It would be interesting to check the behavior on x11 to check if this is ozone-wayland specific

**Comment by michaelforney** [comment]
I dug into this a little bit.

It is because in `DesktopWindowTreeHostOzone::DispatchEvent`, you call `SendEventToProcessor(event)` for key events. `DesktopWindowTreeHostX11` does not. I believe this was just an error in commit 25315fc81890a54e49ea357b4ed0391eff868eb6.

Note in https://codereview.chromium.org/1236923003/diff/140001/ui/views/widget/desktop_aura/desktop_window_tree_host_x11.cc that they not only started calling `GetInputMethod()->DispatchKeyEvent`, but they also stopped calling `SendEventToProcessor` for key events.

**Comment by kalyankondapally** [comment]
Thx for looking into this. Will check this, do you have a patch already ?

**Comment by jbrianceau** [comment]
Seems to be the same problem I bumped into in https://github.com/01org/ozone-wayland/issues/453
So this issue and https://github.com/01org/ozone-wayland/issues/453 are very likely to be duplicates.

**Comment by liveopp** [comment]
Is this fixed?

**Comment by jbrianceau** [comment]
As far as I know, it is not fixed yet @liveopp 
Is this patch fixing the issue for you : https://github.com/jbrianceau/ozone-wayland/commit/ec897cd2e98722aa143166ff36c214214a238149 ?

**Comment by jbrianceau** [comment]
Should be fixed in master branch now : https://github.com/01org/ozone-wayland/commit/607abb6913f761e3cb7dfccfd72e30173e4bc4ea
